src/analysis/retail/paladin/holy/modules/heroTalents/herald/Aurora.tsx

```diff
@@ -0,0 +1,150 @@
+    );
+    this.addEventListener(
+      Events.heal.by(SELECTED_PLAYER).spell(HEALING_HOLY_POWER_SPENDERS),
+      this.holyPowerHeal,
+    );
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.DIVINE_PURPOSE_BUFF),
+      this.applyBuff,
+    );
+    this.addEventListener(
+      Events.refreshbuff.by(SELECTED_PLAYER).spell(SPELLS.DIVINE_PURPOSE_BUFF),
+      this.refreshBuff,
+    );
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.DIVINE_PURPOSE_BUFF),
+      this.removeBuff,
+    );
+  }
+
+  holyPowerDamage(event: DamageEvent) {
+    if (this.hasProc || this.buffRemovedTimestamp + TRACK_BUFFER > event.timestamp) {
+      this.damageDone += (event.amount || 0) + (event.absorbed || 0);
+    }
+  }
+
+  holyPowerHeal(event: HealEvent) {
+    if (this.hasProc || this.buffRemovedTimestamp + TRACK_BUFFER > event.timestamp) {
+      this.healingDone += (event.amount || 0) + (event.absorbed || 0);
+      this.overhealingDone += event.overheal || 0;
+    }
+  }
+
+  applyBuff(event: ApplyBuffEvent | RefreshBuffEvent) {
+    if (GetRelatedEvent(event, AURORA_DIVINE_PURPOSE)) {
+      this.hasProc = true;
+      this.procsGained += 1;
+      this.buffAppliedTimestamp = event.timestamp;
+    }
+  }
+
+  refreshBuff(event: ApplyBuffEvent | RefreshBuffEvent) {
+    if (GetRelatedEvent(event, AURORA_DIVINE_PURPOSE)) {
+      this.hasProc = true;
+      this.procsGained += 1;
+      this.buffAppliedTimestamp = event.timestamp;
+      this.procsWasted += 1;
+    }
+  }
+
+  removeBuff(event: RemoveBuffEvent) {
+    if (this.hasProc) {
+      const lowerRoughTime = this.buffAppliedTimestamp + BUFF_TIME;
+      if (lowerRoughTime < event.timestamp) {
+        this.procsWasted += 1;
+      }
+      this.averageTimeTillBuffConsumed += event.timestamp - this.buffAppliedTimestamp;
+      this.buffRemovedTimestamp = event.timestamp;
+      this.hasProc = false;
+    }
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        position={STATISTIC_ORDER.CORE(11)}
+        size="flexible"
+        category={STATISTIC_CATEGORY.HERO_TALENTS}
+        tooltip={
+          <>
+            <ul>
+              <li>
+                Average Time Till Buff Consumed:{' '}
+                {formatDuration(this.averageTimeTillBuffConsumed / this.procsGained)}
+              </li>
+              <li>Total Buffs: {this.procsGained}</li>
+              <li>Wasted Buffs: {this.procsWasted}</li>
+              <li>Damage: {formatNumber(this.damageDone)}</li>
+              <li>Healing: {formatNumber(this.healingDone)}</li>
+              <li>Overhealing: {formatNumber(this.overhealingDone)}</li>
+            </ul>
+          </>
+        }
+      >
+        <TalentSpellText talent={TALENTS.AURORA_TALENT}>
+          <ItemHealingDone amount={this.healingDone} /> <br />
+          {this.damageDone > 0 && (
+            <>
+              <ItemDamageDone amount={this.damageDone} /> <br />
```

Comment:
Avoid using `br` and prefer using a `div` instead as it plays better on all devices
```suggestion
          <div><ItemHealingDone amount={this.healingDone} /></div>
          {this.damageDone > 0 && (
            <>
              <div><ItemDamageDone amount={this.damageDone} /></div>
```